tdrs-backend/tdpservice/data_files/admin.py

```diff
@@ -7,18 +7,19 @@
 class DataFileAdmin(admin.ModelAdmin):
     """Enforce read-only on the Data File admin form."""
 
-    def has_add_permission(self, request):
-        """Deny the user permission to create Data Files in Django Admin."""
-        return False
+    list_display = [
```

Comment:
The removed code was now unnecessary similar to the other admin entry above. The `list_display` and `list_filter` declarations were just added to clean up the display of the Data Files list page in the admin panel.